google/earthenterprise
# Background color fix for jpgs in wms

Verification steps:
1. Publish a globe with "wms" enabled in the admin panel
2. Using qgis or a script, view the wms content..  In qgis try connecting using jpg
3. Background is no longer black by default
